sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RequestData.cs

```diff
@@ -41,6 +41,10 @@ public RequestData(int version, Activity activity, ref ActivityTagsProcessor act
 
             TraceHelper.AddActivityLinksToProperties(activity, ref activityTagsProcessor.UnMappedTags);
             TraceHelper.AddPropertiesToTelemetry(Properties, ref activityTagsProcessor.UnMappedTags);
+            if (activity.Kind == ActivityKind.Consumer)
+            {
+                TraceHelper.AddEnqueuedTimeToMeasurements(activity, Measurements);
```

Comment:
the Measurements were never used until now, and this is introducing its for the 1st time?

Reply:
Yes